src/plots/cartesian/layout_attributes.js

```diff
@@ -279,22 +279,22 @@ module.exports = {
         pattern: {
             valType: 'enumerated',
             // TODO could add '%H:%M:%S'
-            values: ['%w', '%H', ''],
+            values: ['day of week', 'hour', ''],
             dflt: '',
             role: 'info',
             editType: 'calc',
             description: [
                 'Determines a pattern on the time line that generates breaks.',
-                'If *%w* - Sunday-based weekday as a decimal number [0, 6].',
-                'If *%H* - hour (24-hour clock) as a decimal number [0, 23].',
-                'These are the same directive as in `tickformat`, see',
-                'https://github.com/d3/d3-time-format#locale_format',
+                'If *day of week* - Sunday-based weekday as a decimal number [0, 6].',
+                'If *hour* - hour (24-hour clock) as a decimal number [0, 23].',
+                '*day of week* and *hour* are similar to *%w* and *%H* directives',
+                'applied in `tickformat`, see https://github.com/d3/d3-time-format#locale_format',
                 'for more info.',
                 'Examples:',
-                '- { pattern: \'%w\', bounds: [6, 0], operation: \'[]\' }',
+                '- { pattern: \'day of week\', bounds: [6, 0] }',
```

Comment:
Does this PR implement the default operation?

Reply:
No. Those changes would be done in a separate PR.